modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/query/ElasticsearchQueryTranslatorTest.java

```diff
@@ -112,13 +124,38 @@ public void testTranslateInnerBoostBooleanQuery() {
 	}
 
 	@Test
-	public void testTranslateTermsQueryExceedingMaxAllowedTerms()
-		throws Exception {
+	public void testTranslateTermsFilterExceedingMaxAllowedTerms() {
+		TermsFilter termsFilter = new TermsFilter("groupId");
+
+		termsFilter.addValues("0", "1", "2", "3", "4", "5", "6", "7", "8", "9");
+
+		Integer defaultMaxTermsCount = ReflectionTestUtil.getFieldValue(
```

Comment:
There is no any other variable with "maxTermsCount" in its name. So change it to "maxTermsCount".